metrics/web/lib/common/presentation/navigation/widgets/page_parameters_proxy.dart

```diff
@@ -0,0 +1,78 @@
+import 'package:flutter/material.dart';
+import 'package:metrics/common/presentation/navigation/models/page_parameters_model.dart';
+import 'package:metrics/common/presentation/navigation/state/navigation_notifier.dart';
+import 'package:metrics/common/presentation/state/page_notifier.dart';
+import 'package:provider/provider.dart';
+
+/// A [Widget] that purposes to handle a connection between
+/// the [NavigationNotifier] and a specific [PageNotifier].
+class PageParametersProxy extends StatefulWidget {
+  /// The [PageNotifier] that provides an ability to handle page parameters.
+  final PageNotifier pageNotifier;
```

Comment:
📱 
Should this widget take a list of `PageNotifier`s to be able to subscribe to all of them at once? What do you think?